x-pack/plugins/xpack_main/public/components/telemetry/opt_in_details_component.js

```diff
@@ -66,6 +80,27 @@ export class OptInExampleFlyout extends Component {
       );
     }
 
+    if (unprivilegedUser) {
```

Comment:
does it make sense to declare explicitly that user should have privileges to perform some type of actions? like `hasPrivilegesToRead`